testsuite/MDAnalysisTests/coordinates/base.py

```diff
@@ -445,6 +446,29 @@ def test_pickle_reader(self, reader):
         assert_equal(reader.ts, reader_p.ts,
                      "Timestep is changed after pickling")
 
+    
+    @pytest.mark.parametrize('order', ('fac', 'fca', 'afc', 'acf', 'caf', 'cfa'))
+    def test_timeseries_shape(self, reader, order):
+        timeseries = reader.timeseries(order=order)
+        a_index = order.find('a')
+        f_index = order.find('f')
+        c_index = order.find('c')
```

Comment:
```suggestion
        a_index = order.index('a')
        f_index = order.index('f')
        c_index = order.index('c')
```
It's a trivial thing, but I'd recommend index over find because the latter returns -1 on failure instead of a ValueError, which can lead to some unintended behaviour.